cfgov/jinja2/v1/_includes/organisms/data_snapshot.html

```diff
@@ -65,21 +67,25 @@ <h3><b>{{ value.inquiry_year_over_year_change }}</b>
                  alt="{{ image_alt_value(media) }}">
         </div>
 
-        {% if value.denial_year_over_year_change %}
+        {% if value.tightness_year_over_year_change %}
         <br/>
-        <h3><b>{{ value.denial_year_over_year_change }}</b>
+        <h3><b>{{ value.tightness_year_over_year_change }}</b>
```

Comment:
Please remove the `<b>` and `</b>`. We don't want the extra boldness here, or on any H3 tag. It makes the tag too heavy.

Reply:
aha! Can do.